src/Patchwork/Utf8/Bootup.php

```diff
@@ -42,6 +42,11 @@ public static function initMbstring()
                 user_error('php.ini settings: Please disable mbstring.encoding_translation or set mbstring.http_input to "pass"',  E_USER_WARNING);
             }
 
+            // Fix for PHP > 8.0.0
+            if (!defined("MB_OVERLOAD_STRING")) {
+                define("MB_OVERLOAD_STRING", 0);
+            }
+
             if (MB_OVERLOAD_STRING & (int) ini_get('mbstring.func_overload')) {
```

Comment:
Yep, but this will not resolve the issue with php versions > 8.0.0, because the MB_OVERLOAD_STRING is removed in all versions after 8.0.0. So its not defined as a constant. Also if there is another file in which this constant is used, it will trow an error tehre too. :)

ref: https://www.php.net/manual/en/mbstring.constants.php